stable/concourse/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: concourse
-version: 0.10.7
-appVersion: 3.6.0
+version: 0.11.0
+appVersion: 3.8.0
```

Comment:
The image tag is `3.5.0`. The apps version isn't 3.8.0. This needs to be cleaned up

Reply:
Addressed in https://github.com/kubernetes/charts/pull/3203/files#diff-969838d261afed87173f82e43073caefR71 and https://github.com/kubernetes/charts/pull/3203/files#diff-95cc28250f44d26990fee96ad2fbd63dR16